planner_cspace/src/planner_3d.cpp

```diff
@@ -1299,6 +1302,11 @@ class Planner3dNode
       {
         updateStart();
 
+        if (ros::Time::now() > next_replan_time)
+        {
+          return;
+        }
+
```

Comment:
This block seems have to be reverted and placed after `if (has_map_){}` block.
I guess it adds some delay and increases test failure rate.

See https://github.com/at-wat/neonavigation/pull/484